src/stravalib/client.py

```diff
@@ -67,6 +67,8 @@ def __init__(
         rate_limit_requests: bool = True,
         rate_limiter: limiter.RateLimiter | None = None,
         requests_session: Session | None = None,
+        token_expires: int | None = None,
+        token_refresh: str | None = None,
```

Comment:
Let's rename this throughout to `refresh_token` to retain consistency with Strava's naming and our naming of `access_token`.

Reply:
I've done this throughout! Are you comfortable with keeping `token_expires` as that makes it a verb?